### DISCUSSION TITLE
❌ The feature permission policy header

### DISCUSSION THREAD

**Discussion by ecki** [oshp/oshp-tracking#10] [Q&A]
The best practice shows a quite long feature-policy header. I think that has 3 problems:

a) it’s long
B) it might become incomplete
C) i have the feeling, it does somewhat not really fit into the intended use (non of the W3C examples go so far)

but maybe I am wrong about point c, for that reason I started an issue in the W3C reference project, maybe you would like to visit:

https://github.com/w3c/webappsec-permissions-policy/issues/483

**Comment by righettod** [comment]
Hi,

I totally share your remark 😃 

I proposed such configuration to **deny all by default** but it's true that it cause the value to be long.

I will review my proposal by deep diving into the header spec (+ tests on browsers) to address the concern.

Thanks again for you very useful remarks and the issue opened on W3C repository with cross reference to OSHP 👍

**Comment by righettod** [comment]
⚒ Issue #11 created to track the work on this topic.

**Comment by righettod** [comment]
Once the issue https://github.com/w3c/webappsec-permissions-policy/issues/483 will have a feedback from the project team, then, I will adapt the policy to achieve the goal of *off by default* as @ecki mentioned.

Currently, I have read the spec another time and I can do nothing more than explicitly mention all directives to empty allow list with the risk of forgot new directives 👎

**Comment by righettod** [comment]
Issue still open 😞 

![image](https://github.com/oshp/oshp-tracking/assets/1573775/a5bdf9a3-f964-4b44-978c-faaee8e81bfb)

**Comment by righettod** [comment]
I close the topic because the [issue](https://github.com/w3c/webappsec-permissions-policy/issues/483) never get reply 😞
